tasmoadmin/src/Helper/TasmotaHelper.php

```diff
@@ -86,8 +88,34 @@ public function getReleases(): array
 
     public function getLatestFirmwares(string $configuredFirmware): AutoFirmwareResult
     {
-        $firmwareResult = $this->getLatestRelease();
+        $isEsp32 = str_starts_with($configuredFirmware, 'tasmota32');
```

Comment:
This assumes that if the firmware starts with this it's esp32 - not sure it's the best thing to assume, but seems to work :/